projects/core/src/personalization/http-interceptors/occ-personalization-id.interceptor.ts

```diff
@@ -20,24 +20,31 @@ const PERSONALIZATION_ID_KEY = 'personalization-id';
 export class OccPersonalizationIdInterceptor implements HttpInterceptor {
   private personalizationId: string;
   private requestHeader: string;
+  private enabled: boolean;
 
   constructor(
     private config: PersonalizationConfig,
     private occEndpoints: OccEndpointsService,
     private winRef: WindowRef,
     @Inject(PLATFORM_ID) private platform: any
   ) {
-    this.requestHeader = this.config.personalization.httpHeaderName.id.toLowerCase();
-    this.personalizationId =
-      this.winRef.localStorage &&
-      this.winRef.localStorage.getItem(PERSONALIZATION_ID_KEY);
+    this.enabled = this.config.personalization.enabled || false;
```

Comment:
Why `|| false`?

Reply:
If there is no value in `this.config.personalization.enabled`, we want `this.enabled` is `false`.